webpack.config.js

```diff
@@ -83,27 +72,27 @@ if (process.env.NODE_ENV !== 'development') {
       new webpack.DefinePlugin({
         loadA11y: process.env.loadA11y || false,
         nodeEnv: JSON.stringify('development'),
+        'process.env': {
+          NODE_ENV: JSON.stringify('production'),
```

Comment:
I think it has assigned `NODE ENV` on Line 75. And I think the value should be 'development' as line 7 indicates.

Reply:
Yeah, it's not needed here. I added it accidentally working on a different PR. It's needed when using webpack/babel to build but we don't use webpack/babel to build right now. But it's part of #4.